src/scalper.ts

```diff
@@ -795,3 +803,16 @@ function waitForFill(conditionFunction) {
   };
   return new Promise(poll);
 }
+
+function waitForGamma(conditionFunction) {
```

Comment:
can you put a comment above this to explain what it is doing? Seems like it is waiting for either all the gamma scalps to happen, a sufficient amount of polls?